bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -1043,6 +1045,7 @@ void ReplicaImp::onMessage<PrePrepareMsg>(PrePrepareMsg *msg) {
     LOG_INFO(GL,
              "Ignoring PrePrepareMsg because system is stopped at checkpoint pending control state operation (upgrade, "
              "etc...)");
+    // TODO(LG): we should delete the message.
```

Comment:
Then why don't we do it?

Reply:
You're right it shouldn't be in this PR - removed.